tests/integration/auth.spec.coffee

```diff
@@ -30,8 +30,10 @@ describe 'SDK authentication', ->
 		describe 'resin.auth.authenticate()', ->
 
 			it 'should eventually be a valid token given valid credentials', ->
-				resin.auth.authenticate(credentials).then(resin.token.isValid).then (isValid) ->
-					m.chai.expect(isValid).to.be.true
+				resin.auth.authenticate(credentials)
+				.then(resin.auth.loginWithToken)
+				.then(resin.auth.getKey).then (key) ->
```

Comment:
Ultra-nit: I think this `then` should be on the next line, so it matches the others